Document/0x06a-Platform-Overview.md

```diff
@@ -161,9 +162,9 @@ $ ipainstaller App_name.ipa
 
 #### App Permissions
 
-In contrast to Android, iOS apps do not have preassigned permissions. Instead, the user is asked to grant permission during runtime when an app attempts to use a sensitive API for the first time. Once the app has asked for a permission, it is listed in the Settings > Privacy menu, allowing the user to modify the app-specific setting. Apple calls this permission concept [privacy controls](https://support.apple.com/en-sg/HT203033 "Apple - About privacy and Location Services in iOS 8 and later").
+In contrast to Android, iOS applications do not have preassigned permissions. Instead, the user is asked to grant permission during the runtime when the app attempts to use a sensitive API for the first time. Once the app has asked for a permission, it is listed in the Settings > Privacy menu, allowing the user to modify the app-specific setting. Apple calls this permission concept [privacy controls](https://support.apple.com/en-sg/HT203033 "Apple - About privacy and Location Services in iOS 8 and later").

```

Comment:
> during the runtime

I think "during runtime" or "at runtime" is correct here, without "the". In development, compile-time indicates "during compilation", and run-time indicates "during the running of the application".